chore(files): update to files css to make things line up nicer

WanderingHogan opened the pull request:
<!--  Thanks for sending a pull request!-->

### What this PR does 📖
Minor changes to CSS to fix issue where flexbox was vertically distributing files
- 

### Which issue(s) this PR fixes 🔨

- Resolve #322

<!--Add the ticket Github number such as #Resolve #001 to automatically link the PR to the issue-->

### Special notes for reviewers 🗒️
It looks like I changed a whole bunch in storage.rs, but I just put the files content inside another span so I could apply different flexbox css to a parent container to the individual icons

### Additional comments 🎤

phillsatellite commented:
@WanderingHogan  I had some issues still on Windows and MacOS (Windows seemed to work a lot better though)

MacOS:
On Mac when uploading files the first 7 looked good but once I uploaded the 8th and it created a second line thats when things would go off grid 

https://user-images.githubusercontent.com/93608357/221951881-cec8ea23-737b-4424-a7f9-9f4891446f09.mov

<img width="956" alt="Screenshot 2023-02-28 at 1 50 26 PM" src="https://user-images.githubusercontent.com/93608357/221951895-b802eb2e-9f07-4c6c-a124-dffd110a0e1d.png">

Windows:
On Windows it actually lined up very well it was just when I created a folder, it would then throw everything off grid 

![image](https://user-images.githubusercontent.com/93608357/221952093-5a41458f-e129-427c-9a27-28d5c43490f0.png)

WanderingHogan committed:
chore(files): update to files css to make things line up nicer

WanderingHogan commented:
Updates:

- Took out the function that adds the ellipses in the middle (like on Mac) so we can just use CSS and put ellipses at the end (like on windows) which is the only behavior available in pure css
- Added a hover to show more of the title, and a title attribute to show the whole un-truncated title
- fixed alignment issues between folders and files
- added a label for file type since it's no longer at the end of the string (only if cant add a thumbnail)

WanderingHogan committed:
remove commented out lines

InfamousVague committed:
Merge branch 'dev' into files-css
